server/config/persist_options.go

```diff
@@ -330,13 +338,18 @@ func (o *PersistOptions) GetStoreLimit(storeID uint64) (returnSC StoreLimitConfi
 		RemovePeer: DefaultStoreLimit.GetDefaultStoreLimit(storelimit.RemovePeer),
 	}
 	if ok1 || ok2 {
-		r, ok := v1.(float64)
-		if ok {
+		r, err := strconv.ParseFloat(v1, 64)
```

Comment:
How about abstracting this logic into a function?

Reply:
Yes, I plan to do it.